src/test/java/org/apache/commons/validator/routines/checkdigit/AbstractCheckDigitTest.java

```diff
@@ -154,7 +154,12 @@ public void testCalculateInvalid() {
                     log.debug("   " + i + " Testing Invalid Check Digit, Code=[" + code + "]");
                 }
                 final String expected = checkDigit(code);
-                final String actual = routine.calculate(removeCheckDigit(code));
+                String codeWithNoCheckDigit = removeCheckDigit(code);
+                if(codeWithNoCheckDigit==null) {
```

Comment:
1. Use spaces around '=='.
2. Rebase on git master to pick up a Checkstyle rule that will check for this.
3. Run `mvn` by itself to check